pollux/lib/core/src/main/scala/io/iohk/atala/pollux/core/service/serdes/credentialdefinition/PrivateV1.scala

```diff
@@ -0,0 +1,83 @@
+package io.iohk.atala.pollux.core.service.serdes.credentialdefinition
```

Comment:
I would move the whole `credentialdefinition` package under the sibling `anoncreds` one, so the user knows the credential definition type we are talking about by the package name.

I'm even wondering if we shouldn't keep `AnonCredsXxx` as a prefix for those classes and move them directly under the `anoncreds` package. This will notably prevent import conflicts, knowing that we will soon implement Revocation Registry Definitions, which also have a public and a private part.

Reply:
I moved CredentialDefinition as suggested. Regarding adding AnonCredsXXX prefix back. We can do it once we have the conflict with revocation.